modules/nf-core/limma/differential/templates/limma_de.R

```diff
@@ -393,34 +427,49 @@ fit <- do.call(lmFit, lmfit_args)
 # Contrasts bit
 
 # Create the contrast string for the specified comparison
+if (!is.null(opt\$contrast_string)) {
+    cat("Using contrast string:", opt\$contrast_string, "\n")
+    contrast_string <- as.character(opt\$contrast_string)
+
+    # Display design matrix
+    cat("Design matrix:")
+
+    head(design, 3)
+    print(colnames(design))
```

Comment:
Is this a debug statement of part of the logging information? 
If the latter, wouldn't `head` need a print/cat?